<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(13, 18, 31);
color: rgb(220, 220, 220);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="title_widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 63, 65);
background-color: rgb(20, 28, 36);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="title_label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>TitleLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_title">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_min">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>min</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_max">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>max</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_close">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="menu_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_about">
         <property name="text">
          <string>about</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_menu">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="main_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QDockWidget" name="dockWidget_left">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>524287</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 63, 65);
background-color: rgb(20, 28, 36);</string>
         </property>
         <widget class="QWidget" name="dockWidgetContents_left"/>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_1">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QCustomPlot" name="widget_plot" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <widget class="QLabel" name="label_page_2">
           <property name="geometry">
            <rect>
             <x>500</x>
             <y>270</y>
             <width>591</width>
             <height>151</height>
            </rect>
           </property>
           <property name="text">
            <string>label_page_2</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QDockWidget" name="dockWidget_ctrl">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>524287</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 63, 65);
background-color: rgb(20, 28, 36);</string>
         </property>
         <property name="windowTitle">
          <string>控制面板</string>
         </property>
         <widget class="QWidget" name="dockWidgetContents_ctrl">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>15</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_ctrl">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_devices">
               <item>
                <widget class="QLabel" name="label_devices">
                 <property name="text">
                  <string>设备选择</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_devices">
                 <item>
                  <property name="text">
                   <string>bb60c</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_freq">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>中心频点(MHz)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox_freq">
                 <property name="maximum">
                  <double>100000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_bw">
               <item>
                <widget class="QLabel" name="label_bw">
                 <property name="text">
                  <string>带宽(Mhz)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox_bw">
                 <property name="maximum">
                  <double>100000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_rbw">
               <item>
                <widget class="QLabel" name="label_rbw">
                 <property name="text">
                  <string>RBW(kHz)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox_rbw">
                 <property name="maximum">
                  <double>100000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_vbw">
               <item>
                <widget class="QLabel" name="label_vbw">
                 <property name="text">
                  <string>VBW(kHz)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox_vbw">
                 <property name="maximum">
                  <double>100000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_start_btn">
               <item>
                <spacer name="horizontalLayout_start_btn_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btn_start">
                 <property name="text">
                  <string>   START   </string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalLayout_start_btn_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_ctrl">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
